stepup/tests/behat/features/ra_login-exception.feature

```diff
@@ -12,3 +12,31 @@ Feature: A RAA can only manage R RA(A)'s on the promotion page
     Then I should see "Error - Access denied"
      And I should see "Authentication was successful, but you are not authorised to use the RA management portal"
 
+  Scenario: User "jane-d4" tries to login with no 2FA token available
+    Given a user "jane-d4" identified by "urn:collab:person:institution-d.example.com:jane-d4" from institution "institution-d.example.com"
+    # The identity does not have a second factor token and can by no means log in to RA
+    And I try to login into the ra portal as "jane-d4"
+    And I press "Submit"
+    Then I should see "Error - Not authorised to sign in"
+    And I should see "Error code"
+    And I should see "11430"
+
+  Scenario: User "joe-a3" tries to login while no acceptable 2FA token is available
+    Given a user "joe-a3" identified by "urn:collab:person:institution-a.example.com:joe-a3" from institution "institution-a.example.com"
```

Comment:
Arguably, this scenario looks a lot like the previous one.. But it is somewhat different as this  time the idenitty does have a token (but it is not suitable)